### PR TITLE
Ask contributors to provide estimates when fixing issues

### PR DESCRIPTION
**Pull request by mmusgrov**
The PR includes:

- guidelines for estimating bug fixes (I didn't create a discussion for this item since it is not mandatory, although any engineer can provide estimates)
- license header fix
- minor formatting fix

!LRA !CORE !TOMCAT !AS_TESTS !RTS !JACOCO !XTS !QA_JTA !QA_JTS_OPENJDKORB !PERFORMANCE !DB_TESTS !mysql !db2 !postgres !oracle

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,26 +30,31 @@ New files MUST include the project SPDX License Identifier (Apache-2.0), for exa
...
+- *Fix Version/s*: Leave this field blank, the engineer who fixes the issue will set the correct version when the PR is merged.
+- *Component/s*: The components relevant to the problem. Leave the field blank if you're not sure which components are affected.
 
 Don't forget to indicate which version of Narayana, Java and Maven you are using.
 
+Additionally, for more involved items of work, if you are able to estimate the work required to fix the issue or RFE please provide, as best you can, details using the following fields:
+
+- *Original and Remaining Estimate* for estimating resources
+- *Estimated Difficulty* for estimating complexity
+
```
Comment (jmfinelli) [inline CONTRIBUTING.md:57]:
```suggestion
Additionally, for more involved items of work, if you are able to estimate the work required to fix the issue or RFE please provide, as best you can, details using the following fields of the JBTM template:

- *Original and Remaining Estimate* for estimating resources
- *Estimated Difficulty* for estimating complexity

```

Comment (mmusgrov) [reply CONTRIBUTING.md:57]: What is "the JBTM template"?

Comment (jmfinelli) [reply CONTRIBUTING.md:57]: Sorry, I don't have a better way to describe it :-( I meant the form that gets opened when creating a new JBTM. In there, we have `Original Estimate`, `Remaining Estimate` and `Estimated Difficulty`.

Comment (marcosgopen) [reply CONTRIBUTING.md:57]: I see, you mean [this](https://issues.redhat.com/secure/CreateIssue.jspa) form.

Comment (jmfinelli) [reply CONTRIBUTING.md:57]: > I see, you mean [this](https://issues.redhat.com/secure/CreateIssue.jspa) form.

Yep, that's it :-) Thanks Marco

Comment (mmusgrov) [reply CONTRIBUTING.md:57]: If you can't describe it easily and if it's not common terminology then it shouldn't be in an introductory document.

**Review by jmfinelli — APPROVED** [review]

**Pull request merged**